src/snowflake/cli/plugins/snowpark/commands.py

```diff
@@ -363,11 +380,15 @@ def build(
             requirements = package_utils.parse_requirements(
                 requirements_file=snowpark_paths.defined_requirements_file,
             )
+            anaconda: Optional[AnacondaChannel] = (
+                None if ignore_anaconda else AnacondaChannel.from_snowflake()
+            )
 
             download_result = package_utils.download_unavailable_packages(
                 requirements=requirements,
                 target_dir=packages_dir,
                 ignore_anaconda=ignore_anaconda,
+                anaconda=anaconda,
```

Comment:
Does this method requires anaconda or packages available in anaconda? If the latter is true the we can do something like
```
anaconda_packages = [] if ignore_anaconda else AnacondaChannel.from_snowflake().packages

download_result = package_utils.download_unavailable_packages(anaconda_packages=anaconda_packages, ...)
```

In this way we have clear interface. WDYT?

Reply:
Refactored - I've added `AnacondaChannel.empty()`, which ignores all packages. This way `download_unavailable_packages` can forget about `ignore_anacoda=true` case

Reply:
Nice and clean 🚀